apps/examples/wifi_manager/wm_test/wm_test.c

```diff
@@ -650,6 +676,14 @@ int _wt_parse_stress(struct wt_options *opt, int argc, char *argv[])
 		}
 	}
 
+	if (!strncmp(argv[4], "file", 4)) {
```

Comment:
Can we change this please ? (!strncmp(argv[4], "file", 5)

Reply:
Sir, changes done as suggested.